app/src/main/java/in/co/rajkumaar/amritarepo/activities/LaunchingActivity.java

```diff
@@ -361,16 +375,19 @@ public void onClick(DialogInterface dialog, int item) {
 
     @Override
     protected void onPostResume() {
-        if (Utils.isConnected(LaunchingActivity.this) && !BuildConfig.DEBUG) {
-            checkUpdate();
-        }
         super.onPostResume();
+        if (android.os.Build.VERSION.SDK_INT < android.os.Build.VERSION_CODES.LOLLIPOP){
```

Comment:
Please completely remove this. The `checkUpdate()` works by fetching the version from Firebase, where in I've to manually update everytime I make a release. It's a pain.
So, as we discussed, set the minSdk to 21 and remove all these if conditions and remove everything related to `checkUpdate()`.